onnx/defs/traditionalml/defs.cc

```diff
@@ -28,10 +28,10 @@ ONNX_ML_OPERATOR_SET_SCHEMA(
              "tensor(int64)",
              "tensor(int32)",
              "tensor(string)"},
-            "allowed types."));
+            ""));
 
 static const char* Binarizer_ver1_doc = R"DOC(
-    Makes values 1 or 0 based on a single threshold.
+    Replaces the values of the input tensor by either 0 or 1, element-wise, based on the outcome of a comparison against a threshold value.
```

Comment:
>Replaces [](start = 4, length = 8)

I think @  is correct, in which case this should also be set.

Reply:
How about "Maps the values of the input tensor to either 0 or 1 …"?

Reply:
@shauheen, I'm not sure I understand your comment. What are you referring to?

Reply:
I assumed he meant change "Replace" to "set" …